### PR TITLE
Autreg 261 services to add and get uischema

### PR DESCRIPTION
**Pull request by Eiriknil**

### CODE REVIEW

File: api/src/main/java/no/bibsys/web/DatabaseResource.java [hunk 1]
```diff
@@ -198,42 +201,73 @@ public Response getRegistrySchema(@HeaderParam(ApiKeyConstants.API_KEY_PARAM_NAM
...
+    public Response updateRegistryUiSchema(@HeaderParam(ApiKeyConstants.API_KEY_PARAM_NAME) String apiKey,
                                          @Parameter(in = ParameterIn.PATH, name = REGISTRY_NAME, required = true,
                                                  description = NAME_OF_REGISTRY_TO + "update",
                                                  schema = @Schema(type = STRING)) @PathParam(REGISTRY_NAME)
-                                                 String registryName, @RequestBody(description = "Validation schema",
-            content = @Content(schema = @Schema(type = STRING))) String schema)
+                                                 String registryName, @RequestBody(description = "uischema",
+            content = @Content(schema = @Schema(type = STRING))) String uiSchema)
 
             throws IOException, ShaclModelValidationException, TargetClassPropertyObjectIsNotAResourceException {
```
Comment (brinxmat) [inline api/src/main/java/no/bibsys/web/DatabaseResource.java]: No ShaclModelValidationException, I hope :)

File: validation/src/test/java/no/bibsys/entitydata/validation/DataValidatorTest.java [hunk 2]
```diff
@@ -16,17 +22,13 @@
...
-import static org.hamcrest.Matchers.containsString;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import no.bibsys.entitydata.validation.exceptions.EntityFailedShaclValidationException;
+import no.bibsys.utils.IoUtils;
+import no.bibsys.utils.ModelParser;
 
 public class DataValidatorTest extends ModelParser {
 
+    private static final String DELIMITER = "";
```
Comment (brinxmat) [inline validation/src/test/java/no/bibsys/entitydata/validation/DataValidatorTest.java:31]: This is a bit weird, and should at least be EMPTY_STRING = ""; DELIMITER = EMPTY_STRING;

File: validation/src/test/java/no/bibsys/entitydata/validation/DataValidatorTest.java [hunk 3]
```diff
@@ -42,7 +44,7 @@
             "invalid_schema_and_invalid_graph_fails_message.txt";
     private static final String PATH_SEPARATOR = "/";
     private static final String SOME_BNODE_AND_A_TURTLE_LINE_ENDING = "SOME_BNODE\" ;";
-    private static final String EMPTY_STRING = "";
+    private static final String EMPTY_STRING = DELIMITER;
     private static final String NEWLINE = "\n";
```
Comment (brinxmat) [inline validation/src/test/java/no/bibsys/entitydata/validation/DataValidatorTest.java:48]: Is NEWLINE always "\n" or System.lineSeparator();

File: api/src/main/java/no/bibsys/web/DatabaseResource.java (hunk 1 above)
Comment (Eiriknil) [reply api/src/main/java/no/bibsys/web/DatabaseResource.java]: Removed

**Pull request merged**